addons/im_livechat/static/src/scss/im_livechat.scss

```diff
@@ -20,7 +20,7 @@
 }
 
 .o_thread_window {
-    z-index: 1002; // to go over the navbar
+    z-index: $zindex-fixed + 5; // to go over the navbar
```

Comment:
Sorry to be annoying but the best way would be like this:
```scss
($zindex-fixed  + $zindex-modal-backdrop) / 2
```
That way, you exactly says what you want: "I want the chat window to be over navbar but below modal backdrop" and you are independant from the z-index increment bootstrap used. Also ideally, you can put that in a separate variable people can use if needed... but not sure about the im_livechat file structure for that and not really required in stable so I let you judge :+1: 

You can also remove the comment now, using the variables is explicit enough.

In fact: https://github.com/odoo/odoo/commit/8b5782e38c1e2b3e9aecfa416a4d21f9d0745304 ... seems like the mail team wants that to be over the modal backdrop and they use a strange "-9" value... might be worth to check with them instead :+1:

Reply:
``($zindex-fixed  + $zindex-modal-backdrop) / 2`` should be good.

@ThanhDodeurOdoo @alexkuhn what do you think?

Reply:
Agree with @qsm-odoo's suggestion.